src/state/user/originCountryStorage.ts

```diff
@@ -0,0 +1,34 @@
+import ms from 'ms'
+
+const originCountryKey = 'origin-country'
```

Comment:
Why do we need the timestamp here? Can't we just have the country?

Reply:
if you _do_ remove the timestamp, keep the stored data as an object, so it is extendable in the future.

Reply:
the timestamp is for cache invalidation